pkg/jobs/jobspb/jobs.proto

```diff
@@ -325,7 +325,12 @@ message BackupDetails {
   // tenants.
   bool include_all_secondary_tenants = 25;
 
-  // NEXT ID: 26;
+  // UpdatesClusterMonitoring indicates whether the backup job should update
```

Comment:
eee. proto in backport. Is it worth checking with SRE to ask if they absolutely need this in 23.1? I'd rather not backport this change.

Reply:
yeah, this is something that needs to be in all of the versions that we support on CC